templates/README.md

```diff
@@ -32,7 +31,7 @@ clusterctl generate cluster ibm-powervs-1 --kubernetes-version v1.21.2 \
 --target-namespace default \
 --control-plane-machine-count=3 \
 --worker-machine-count=1 \
---from ./cluster-template-powervs.yaml
+--flavor=powervs > capi-ibmpowervs-quickstart.yaml
```

Comment:
This is not exposed for end user and lets keep this as is and will be helpful if someone wants to modify the template locally and run the some